fail2ban/client/fail2banserver.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: t -*-
+# vi: set ft=python sts=4 ts=4 sw=4 noet :
+#
+# This file is part of Fail2Ban.
+#
+# Fail2Ban is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# Fail2Ban is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Fail2Ban; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+__author__ = "Fail2Ban Developers"
+__copyright__ = "Copyright (c) 2004-2008 Cyril Jaquier, 2012-2014 Yaroslav Halchenko, 2014-2016 Serg G. Brester"
+__license__ = "GPL"
+
+import os
+import sys
+
+from .fail2bancmdline import Fail2banCmdLine, ServerExecutionException, \
+	logSys, PRODUCTION, exit
+
+SERVER = "fail2ban-server"
+
+##
+# \mainpage Fail2Ban
+#
+# \section Introduction
+#
+class Fail2banServer(Fail2banCmdLine):
+
+	# def __init__(self):
+	# 	Fail2banCmdLine.__init__(self)
+
+	##
+	# Start Fail2Ban server in main thread without fork (direct, it can fork itself in Server if daemon=True).
+	#
+	# Start the Fail2ban server in background/foreground (daemon mode or not).
+
+	@staticmethod
+	def startServerDirect(conf, daemon=True):
+		logSys.debug("-- direct starting of server in %s, deamon: %s", os.getpid(), daemon)
```

Comment:
nope, because the same code will be executed in server also (e.g. starting server in foreground).
Are we arguing about each additionally character now?